lib/response.js

```diff
@@ -158,6 +160,9 @@ res.send = function send(body) {
         if (!this.get('Content-Type')) {
           this.type('bin');
         }
+      } else if (isChunkBlob) {
+        this.set('Content-Type', chunk.type || this.get('Content-Type') || 'application/octet-stream')
```

Comment:
This should use `this.type` to set the type like the other code paths in here.

Also normally none of the others will override the existing set type, but this new path seems to do the opposite, which just feels odd. Any particular reason to break the pattern for this?